Solutions/Salem Virtual Cyber Analyst/Playbooks/SendAlertToSalem/azuredeploy.json

```diff
@@ -0,0 +1,220 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Send-Sentinel-Alerts-to-Salem",
+        "description": "Use this playbook to send Sentinel alerts to Salem Virtual Cyber Analyst",
+        "prerequisites": [
+            "Install Salem from the Azure Marketplace",
+            "Obtain the send key from the Alerts Eventhub in the Salem resrouce gorup"
+        ],
+        "author": {
+            "name": "Salem Cyber"
+        },
+        "releaseNotes": [
+            {
+                "version": "1.0.0",
+                "title": "Send-Sentinel-Alerts-to-Salem",
+                "notes": [ "Initial version" ]
+            }
+        ]
+    },
+    "parameters": {
+        "PlaybookName": {
+            "type": "string",
+            "metadata": {
+                "description": "A Globally unique name for the integration logic app"
+            }
+        },
+        "SalemEventHubConnectionString": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The Connection String from the Salem 'alerts' Event Hub"
+            }
+        },
+        "location": {
```

Comment:
Hello @jonbagg Is this valid to create a location parameter in playbook.
Because armm ttk is failing due to this.

Reply:
This might be a false positive.  The error says:  mainTemplate.json must use the location parameter, not resourceGroup().location or deployment().location (except when used as a default value in the main template).  This is how the template is configured with resourceGroup().location only used as the default value of the parameter in the template.